classes/Parameters/UpgradeConfiguration.php

```diff
@@ -160,6 +190,28 @@ public function shouldUpdateRTLFiles(): bool
         return (bool) $this->get('PS_AUTOUP_UPDATE_RTL_FILES');
     }
 
+    public static function shouldDisabledOverride(): bool
```

Comment:
Let's find another name for this method, to replace the "should" and fix the verb in the past tense.

* `isOverrideAllowed()`
* `canOverrideBeUsed()`
* `isOverridingEnabled()`